parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,102 @@
+_Devices tab on Notehub project showing your Notecard is registered to this project._
+
+If you'd prefer to upload this script using the in-browser Notecard Playground instead of downloading the CLI, connect your Notecarrier to your computer via the USB-A to micro USB connector, pair it via serial port in your browser by clicking the **USB Notecard** button in the [Notecard Playground](https://dev.blues.io/notecard-playground/), and copy/paste the whole config file like in the screenshot below. (Don't forget to replace the placeholder `product` and `sn` variables in the script before copy/pasting.)
+
+![Pasting in the Notecard config script in the in-browser Notecard playground](./readme-notecard-playground-script.png)
+
+Now, click the button a few times and hopefully you'll see the LED light on the board flash and the button press data similar to this screenshot below flowing into your own Notehub project.
+
+![Press that button and watch the LED light up](readme-press-the-button.png)
+
+https://user-images.githubusercontent.com/20400845/197624207-36afcfb9-6307-4dc5-acb9-542bec0140cf.mov
+
+![_button.qo events registering in Notehub when the help request button is pressed on the device](readme-notehub-events.png)
+_The `_button.qo` events are the events we'll want to use to trigger SMS alerts._
+
+### Transform Notehub data with JSONata for Twilio
+
+After button press events are successfully being sent to your Notehub project, they'll need to be transformed and routed out to Twilio to trigger SMS alerts to a user's phone.
+
+> **NOTE:** Although this project is using Twilio for SMS alerts, you can route your button presses to any service you'd like, and set up whatever sort of notification system you have in mind.
+
+Follow the [Twilio starter guide](https://dev.blues.io/guides-and-tutorials/twilio-sms-guide/#configuring-the-route) on the Blues Wireless developer experience site to go through the basics of setting up Twilio account, and a route from Notehub to Twilio, until you reach step 8.
+
+Instead of selecting `twilio.qo` files, you're going to want to route out `_button.qo` events in the **Filters** section.
+
+![Selecting just _button.qo events to route to Twilio from Notehub](readme-button-events.png)
+_Select just the `_button.qo` events to route out of Notehub to Twilio._
+
+Then, use [JSONata](https://dev.blues.io/guides-and-tutorials/notecard-guides/using-jsonata-to-transform-json/) (a lightweight query and transformation language for JSON) to filter down and format the data from a button event before sending it to Twilio.
+
+Twilio expects very specific syntax, and you can add the JSONata expression right at the bottom of the new route page you're working in under the **Data** section.
+
+![JSONata expression to format Notehub event for Twilio SMS alerts in Notehub route](readme-notehub-jsonata.png)
```

Comment:
For perfection, don't forget to update the JSONata code in the screenshot too.

Reply:
Good attention to detail - thank you!